raiden/transfer/channel.py

```diff
@@ -844,6 +844,37 @@ def get_amount_locked(end_state: NettingChannelEndState) -> TokenAmount:
     return total_pending + total_unclaimed + total_unclaimed_onchain
 
 
+def get_batch_unlock_gain(
+        channel_state: NettingChannelState,
+) -> Tuple[TokenAmount, TokenAmount]:
```

Comment:
nitpick: I would suggest to change this to a named tuple and thus control the return values more tightly. Returning tuples like that can end up to bugs due to messing up the order at the point of function call consumption.